src/backend/api/bucket.py

```diff
@@ -345,7 +345,7 @@ async def get_bucket_attributes(
     return res
 
 
-@router.head(
```

Comment:
[HEAD](https://developer.mozilla.org/en-US/docs/Web/HTTP/Methods/HEAD) should not have a body.